overlord/state/export_test.go

```diff
@@ -44,3 +44,16 @@ func MockTaskTimes(t *Task, spawnTime, readyTime time.Time) {
 	t.spawnTime = spawnTime
 	t.readyTime = readyTime
 }
+
+func (s *State) AddWarningFull(message string, lastSeen, lastShown time.Time, deleteAfter, repeatAfter time.Duration) {
```

Comment:
Would it make sense to consume the struct rather than the individual fields here?

Reply:
the Warnings struct members are all private, for now at least